src/js/node/http.ts

```diff
@@ -771,13 +773,19 @@ Object.defineProperty(IncomingMessage.prototype, "connection", {
 
 Object.defineProperty(IncomingMessage.prototype, "statusCode", {
   get() {
-    return this[reqSymbol].status;
+    return this[statusSymbol] ?? this[reqSymbol].status;
+  },
+  set(value) {
+    this[statusSymbol] = value;
   },
 });
 
 Object.defineProperty(IncomingMessage.prototype, "statusMessage", {
   get() {
-    return STATUS_CODES[this[reqSymbol].status];
+    return this[statusMessageSymbol] ?? STATUS_CODES[this[reqSymbol].status];
+  },
+  set(value) {
+    this[statusMessageSymbol] = value;
   },
```

Comment:
For context:

Node http2 uses a symbol called `'state'`:
https://github.com/nodejs/node/blob/main/lib/internal/http2/compat.js#L71

Related to IncomingMessage however Node uses a data property (not accessor):
https://github.com/nodejs/node/blob/main/lib/_http_incoming.js#L88

Reply:
yeah, we should just set the property on the object and not use a getter/setter